db/migrate/20210712223435_add_unique_indexes_to_matviews.rb

```diff
@@ -0,0 +1,6 @@
+class AddUniqueIndexesToMatviews < ActiveRecord::Migration[5.2]
+  def change
+    add_index :reporting_patient_blood_pressures, [:patient_id, :month_date], unique: true, name: "patient_blood_pressures_patient_id_month_date"
+    add_index :reporting_patient_visits, [:patient_id, :month_date], unique: true, name: "patient_visits_patient_id_month_date"
```

Comment:
We should change the order of the multicolumn index to be `month_date` first, and `patient_id` second. This way, this index would be used in queries that filter by `month_date`. See the docs for more details on effect of leading columns in multicolumn indexes: https://www.postgresql.org/docs/10/indexes-multicolumn.html

Reply:
We should also change the index on `patient_states` to be the same way.

Reply:
change ✨